topics/genome-annotation/tutorials/funannotate/tutorial.md

```diff
@@ -0,0 +1,478 @@
+
+**InterProScan** is a tool that analyses each protein sequence from our annotation to determine if they contain one or several of the signatures from InterPro. When a protein contains a known signature, the corresponding functional annotation will be assigned to it by **InterProScan**.
+
+**InterProScan** itself runs multiple applications to search for the signatures in the protein sequences. It is possible to select exactly which ones we want to use when launching the analysis (by default all will be run).
+
+> ### {% icon hands_on %} Hands-on
+>
+> 1. {% tool [InterProScan](toolshed.g2.bx.psu.edu/repos/bgruening/interproscan/interproscan/5.52-86.0+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Protein FASTA File"*: `protein sequences` (output of **Funannotate predict annotation** {% icon tool %})
+>    - *"InterProScan database"*: select the latest version available
+>    - *"Applications to run"*: unselect `PROSITE Profiles`, `PROSITE Pattern`, `SUPERFAMILY`,  (see why below)
+>    - *"Output format"*: `Tab-separated values format (TSV)` and `XML`
+>    - *"Use applications with restricted license, only for non-commercial use?"*: `Yes` (set it to `No` if you run InterProScan for commercial use)
+>
+{: .hands_on}
+
+> ### {% icon comment %} Comments
+>
+> Due to bugs in InterProScan, `PROSITE Profiles`, `PROSITE Pattern` and `SUPERFAMILY` don't work with the current version of InterProScan. We disable it for this tutorial, and hopefully it should be possible to use them in future versions of InterProScan.
+>
+> To speed up the processing by InterProScan during this tutorial, you can also disable `Pfam` and `PANTHER` applications. When analysing real data, it is adviced to keep them enabled.
+>
+> When some applications are disabled, you will of course miss the corresponding results in the output of **InterProScan**.
+{: .comment}
+
+The output of this tool is both a tabular file and an XML file. Both contain the same information, but the tabular one is more readable for a Human: each line represents a gene from our annotation, with the different domains and motifs that were found by InterProScan.
+
+Display the TSV file and explore which kind of results were found by InterProScan.
+
+The XML file will be used in the next step.
+
+# Submission to NCBI
+
+If you plan to submit the final genome sequence and annotation to NCBI, there are a few steps to follow.
+
+In this tutorial we will not perform a real submission, but here's a description of the main steps, and how Funannotate can help you in this process. NCBI provides a complete documentation for [genome and annotation submission](https://www.ncbi.nlm.nih.gov/genbank/genomesubmit/).
```

Comment:
maybe this [could use a warning box](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#warning-box), just in case, to make it a bit clearer that you should not do this.

 "Please do not submit this genome to NCBI"